definitions/ext-vpc_network/summary_metrics.yml

```diff
@@ -7,6 +7,11 @@ sources:
       from: KFlow
       where: "provider = 'kentik-vpc'"
       eventId: entity.guid
+    aws-kinesis-firehose/vpc-flow-logs:
+      select: uniqueCount(srcaddr)
```

Comment:
same comments from GM re: uniqueness

Reply:
conferred with @jbeveland27 on the side; this is good to stay as-is for now pending more research on users and their feedback